libknary/util.go

```diff
@@ -240,13 +240,17 @@ func UsageStats(version string) bool {
 	return true
 }
 
-func CheckTLSExpiry(domain string) bool {
-	conn, err := tls.Dial("tcp", domain+":443", nil)
+func CheckTLSExpiry(domain string, config *tls.Config) (bool, error) {
+	port := "443"
+	if os.Getenv("TLS_PORT") != "" {
```

Comment:
Can a comment be added her to note this is used for testing (since there's no mention of this enviro variable in the README